<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>串口显示</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="textEdit_showMess"/>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2,1,2">
         <item>
          <widget class="QLabel" name="label_niceText">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>合格数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_niceNum">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_badRateText">
           <property name="text">
            <string>不合格数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_badRate">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
         <item>
          <widget class="QLabel" name="label_imei">
           <property name="text">
            <string>IMEI:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_imei"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,4">
         <item>
          <widget class="QLabel" name="label_csq_show">
           <property name="text">
            <string>CSQ:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_csq"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4">
         <item>
          <widget class="QLabel" name="label_flag">
           <property name="text">
            <string>产测标志:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_flag">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>开始产测</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>该产品已产测</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_serial">
      <property name="title">
       <string>串口</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>端口</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_uartPort_2">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_uartBate_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>230400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>460800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>921600</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_38">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_uartData_2">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_uartStop_2">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_uartOpen_2">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Bell MT&quot;;
color: rgb(0, 170, 0);


</string>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>检测设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_csq">
           <property name="text">
            <string>CSQ范围</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_csq_min"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_csq_max"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_ok">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>70</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>70</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{background-color: rgb(225, 225, 225);border:2px groove gray;border-radius:35px;padding:2px 4px;border-style: outset;}&quot;
&quot;QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}&quot;
&quot;QPushButton:pressed{background-color:rgb(204, 228, 247);border-style: inset;}&quot;</string>
              </property>
              <property name="text">
               <string>测试中</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_update"/>
    <addaction name="action_help"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionsheng">
   <property name="text">
    <string>sheng</string>
   </property>
  </action>
  <action name="action_update">
   <property name="text">
    <string>升级检测</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
